dnase/submit.sh

```diff
@@ -21,12 +21,14 @@ module load samtools/1.12
 module load bcftools/1.12
 module load trimmomatic/0.39
 module load python/3.8.1
+module load R/3.6.3
```

Comment:
Was this a previously unspecified dependency, or did you add some R code?

Reply:
This was an unspecified dependency.
Without R I was getting errors when running the code.

Reply:
Do you happen to remember roughly where the dependency was? I don't see anything but it's hard to grep for "R" :)

Reply:
I took out the load for R and am running the code to see what error I got.
It might be good to purge all the modules before loading to ensure no background-loaded modules when the job is submitted.

Reply:
Exception in thread "main" htsjdk.samtools.SAMException: Unexpected exception executing [Rscript /tmp/<kid>/script981572324027252574.R /tmp/slurm.tmp.map.Bl6_TP53_humanization_47kb_cWZ304-BS10483A.cegsvectors_LPSwapinGreen,cegsvectors_LPSwapinRed,cegsvectors_TP53_humanization_47kb,cegsvectors_pSpCas9GFP,hg38_full,mm10.4850055.1/FCH577GBGXJ.BS10483A_R1R2_000.baseQ.txt /tmp/slurm.tmp.map.Bl6_TP53_humanization_47kb_cWZ304-BS10483A.cegsvectors_LPSwapinGreen,cegsvectors_LPSwapinRed,cegsvectors_TP53_humanization_47kb,cegsvectors_pSpCas9GFP,hg38_full,mm10.4850055.1/FCH577GBGXJ.BS10483A_R1R2_000.baseQ.pdf FCH577GBGXJ.BS10483A_R1R2_000.cegsvectors_LPSwapinGreen.bam BS10483A]
        at htsjdk.samtools.util.ProcessExecutor.execute(ProcessExecutor.java:111)
        at htsjdk.samtools.util.ProcessExecutor.execute(ProcessExecutor.java:87)
        at picard.util.RExecutor.executeFromFile(RExecutor.java:78)
        at picard.util.RExecutor.executeFromClasspath(RExecutor.java:59)
        at picard.analysis.MeanQualityByCycle.finish(MeanQualityByCycle.java:205)
        at picard.analysis.SinglePassSamProgram.makeItSo(SinglePassSamProgram.java:177)
        at picard.analysis.SinglePassSamProgram.doWork(SinglePassSamProgram.java:94)
        at picard.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:305)
        at picard.cmdline.PicardCommandLine.instanceMain(PicardCommandLine.java:103)
        at picard.cmdline.PicardCommandLine.main(PicardCommandLine.java:113)
Caused by: java.io.IOException: Cannot run program "Rscript": error=2, No such file or directory
        at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
        at java.lang.Runtime.exec(Runtime.java:620)
        at java.lang.Runtime.exec(Runtime.java:485)
        at htsjdk.samtools.util.ProcessExecutor.execute(ProcessExecutor.java:102)
        ... 9 more
Caused by: java.io.IOException: error=2, No such file or directory
        at java.lang.UNIXProcess.forkAndExec(Native Method)
        at java.lang.UNIXProcess.<init>(UNIXProcess.java:248)
        at java.lang.ProcessImpl.start(ProcessImpl.java:134)
        at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
        ... 12 more

Reply:
that's helpful, I'll commit this to master separately